.circleci/config.yml

```diff
@@ -565,101 +241,153 @@ jobs:
           path: pkg/
           destination: gem
 
+job_configuration: &job_configuration
+  - &config-2_0
+    <<: *filters_all_branches_and_tags
+    ruby_version: '2.0'
+    image: palazzem/docker-library:ddtrace_rb_2_0_0
+  - &config-2_1
+    <<: *filters_all_branches_and_tags
+    ruby_version: '2.1'
+    image: palazzem/docker-library:ddtrace_rb_2_1_10
+  - &config-2_2
+    <<: *filters_all_branches_and_tags
+    ruby_version: '2.2'
+    image: palazzem/docker-library:ddtrace_rb_2_2_10
+  - &config-2_3
+    <<: *filters_all_branches_and_tags
+    ruby_version: '2.3'
+    image: palazzem/docker-library:ddtrace_rb_2_3_8
+  - &config-2_4
+    <<: *filters_all_branches_and_tags
+    ruby_version: '2.4'
+    image: palazzem/docker-library:ddtrace_rb_2_4_6
+  - &config-2_5
+    <<: *filters_all_branches_and_tags
+    ruby_version: '2.5'
+    image: marcotc/docker-library:ddtrace_rb_2_5_6
+  - &config-2_6
+    <<: *filters_all_branches_and_tags
+    ruby_version: '2.6'
+    image: marcotc/docker-library:ddtrace_rb_2_6_4
```

Comment:
If we standardize our repository, we can omit the full image path and make it parametrized, like `DataDog/ddtrace_rb:<<parameters.ruby_version>>-ci`.